cmd/lint.go

```diff
@@ -182,6 +188,29 @@ func GetLintCommand() *cobra.Command {
 	cmd.Flags().BoolP("silent", "x", false, "Show nothing except the result.")
 	cmd.Flags().StringP("fail-severity", "n", "error", "Results of this level or above will trigger a failure exit code")
 
+	regErr := cmd.RegisterFlagCompletionFunc("category", cobra.FixedCompletions([]string{
+		model.CategoryAll,
+		model.CategoryDescriptions,
+		model.CategoryExamples,
+		model.CategoryInfo,
+		model.CategoryOperations,
+		model.CategorySchemas,
+		model.CategorySecurity,
+		model.CategoryTags,
+		model.CategoryValidation,
+	}, cobra.ShellCompDirectiveNoFileComp))
+	if regErr != nil {
+		panic(regErr)
```

Comment:
I think just panicing on `RegisterFlagCompletionFunc` errors everywhere is ok, because that indicates a programming error that will manifest immediately at start; it's not something related to environment, arguments, or input.